huaweicloud/provider.go

```diff
@@ -1103,6 +1103,7 @@ func Provider() *schema.Provider {
 			"huaweicloud_dataarts_architecture_directory":       dataarts.ResourceArchitectureDirectory(),
 			"huaweicloud_dataarts_architecture_subject":         dataarts.ResourceArchitectureSubject(),
 			"huaweicloud_dataarts_architecture_business_metric": dataarts.ResourceBusinessMetric(),
+			"huaweicloud_dataarts_architecture_process":         dataarts.ResourceProcess(),
```

Comment:
suggest to add Architecture prefix
```suggestion
			"huaweicloud_dataarts_architecture_process":         dataarts.ResourceArchitectureProcess(),
```

Reply:
fixed.